flight/Modules/UAVOMSPBridge/UAVOMSPBridge.c

```diff
@@ -178,7 +185,7 @@ struct msp_bridge {
 #if defined(PIOS_MSP_STACK_SIZE)
 #define STACK_SIZE_BYTES PIOS_MSP_STACK_SIZE
 #else
-#define STACK_SIZE_BYTES 768
+#define STACK_SIZE_BYTES 1500
```

Comment:
Yah, F1 doesn't matter.  Probably can be lowered with more analysis but works OK on F3.